indigo_content_api/v1/urls.py

```diff
@@ -16,18 +17,18 @@
 
     # Document/work media
     # Work publication document
-    re_path(r'^(?P<frbr_uri>[a-z]{2}[-/]\S+?)/media/publication/(?P<filename>.*)$', views.PublishedDocumentMediaView.as_view({'get': 'get_publication_document'}), name='published-document-publication'),
+    re_path(r'^(?P<frbr_uri>(akn/)?[a-z]{2}[-/]\S+?)/media/publication/(?P<filename>.*)$', views.PublishedDocumentMediaViewV1.as_view({'get': 'get_publication_document'}), name='published-document-publication'),
```

Comment:
I *think* that we can remove the akn completely, rather than making it optional. The V1 api shouldn't have `/akn` at the start of the FRBR URIs at all -- if one is provided, it should return a 404 because the document can't be found.

Reply:
ah. in that case, some more work needs to be done.